crates/transaction-pool/src/pool/mod.rs

```diff
@@ -272,7 +272,12 @@ where
         tx: TransactionValidationOutcome<T::Transaction>,
     ) -> PoolResult<TxHash> {
         match tx {
-            TransactionValidationOutcome::Valid { balance, state_nonce, transaction } => {
+            TransactionValidationOutcome::Valid { 
+                balance, 
+                state_nonce, 
+                transaction, 
+                propagate: _,
```

Comment:
now we need to use this here

https://github.com/paradigmxyz/reth/pull/3664/files#diff-227a6ad3d598e9efb8ab0aad3c77333c17d2d7a81460b310a0ed8dea9e37548aR288 instead of using false